yfinance/base.py

```diff
@@ -1471,8 +1473,13 @@ def get_mutualfund_holders(self, proxy=None, as_dict=False):
             return data
 
     def get_info(self, proxy=None) -> dict:
-        self._quote.proxy = proxy
-        data = self._quote.info
+        try:
+            self._quote.proxy = proxy
+            data = self._quote.info
+        except Exception:
+            data = {}
+        data1 = self.get_query1_info()
+        data.update(data1)
```

Comment:
Revert this change. Instead, modify yfinance/scrapers/quote.py::`Quote.info()` to call your new method instead of `_scrape()`

Reply:
modified as suggested.